### DISCUSSION TITLE
How to use the SSL Certificate?

### DISCUSSION THREAD

**Discussion by simdol2** [exadel-inc/CompreFace#697] [Q&A] [answered]
I have installed the CompreFace as a docker container on a CentOS 8 server. The APIs are working well during our testing. The APIs are accessed using port 8000 and using http. I want to secure it using an SSL certificate, say letsencrypt SSL certificate. How do I secure the CompreFace APIs ?

**Comment by pospielov** [comment, marked as answer]
`compreface-fe` container is based on nginx docker image. Please look at this dockerfile:
https://github.com/exadel-inc/CompreFace/blob/master/ui/docker-prod/Dockerfile
So, we copy this config during the build:
https://github.com/exadel-inc/CompreFace/tree/master/ui/nginx
You need to make a volume on `/etc/nginx/` folder of `compreface-fe` container and place your updated nginx config with enabled SSL
